common/service/pldm/pldm_firmware_update.c

```diff
@@ -493,6 +533,7 @@ static uint8_t request_update(void *mctp_inst, uint8_t *buf, uint16_t len, uint8
 	}
 
 	if (current_state != STATE_IDLE) {
+		LOG_ERR("Current mode %d doesn't meet request state %d", current_state, STATE_IDLE);
```

Comment:
Can we improve this error message?  I think the message should be something like "firmware update failed because current state is not idle"